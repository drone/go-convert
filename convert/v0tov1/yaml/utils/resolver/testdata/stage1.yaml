
pipeline:
  stages:
  - template:
      uses: golang
      with:
        version: "latest"
