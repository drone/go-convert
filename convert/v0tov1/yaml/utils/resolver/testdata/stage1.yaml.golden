

pipeline:
  stages:
  - context:
      inputs:
        version: "latest"
    steps:
    - run:
        script: go test ./...
        container:
          image: golang:${{inputs.version}}
    - run:
        script: go build
        container:
          image: golang:${{inputs.version}}
